fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/election/fragments/ElectionSetupFragment.java

```diff
@@ -252,7 +254,7 @@ private void setupElectionSubmitButton() {
                           setCurrentFragment(
                               getParentFragmentManager(),
                               R.id.fragment_lao_detail,
-                              LaoDetailFragment::newInstance),
+                              EventListFragment::newInstance),
```

Comment:
is this fixing the back navigation as well ?

Reply:
No this is just te renaming of the same fragment to something more appropriate.